mcs/class/System.Private.CoreLib/System.Reflection.Emit/ModuleBuilder.cs

```diff
@@ -456,19 +455,15 @@ public override Type GetType (string className, bool throwOnError, bool ignoreCa
 		public void SetCustomAttribute( ConstructorInfo con, byte[] binaryAttribute) {
 			SetCustomAttribute (new CustomAttributeBuilder (con, binaryAttribute));
 		}
-
-		public ISymbolWriter GetSymWriter () {
-			return symbolWriter;
-		}
-
-		public ISymbolDocumentWriter DefineDocument (string url, Guid language, Guid languageVendor, Guid documentType)
+/*
```

Comment:
This method does exist and its public in coreclr's SRE code.

Reply:
not really, it's implementation detail check the public API at https://github.com/dotnet/corefx/blob/master/src/System.Reflection.Emit/ref/System.Reflection.Emit.cs#L260